tests/Tests/EntityManagerTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\UnitOfWork;
+use PHPUnit\Framework\TestCase;
+use ReflectionProperty;
+use Symfony\Component\VarExporter\LazyGhostTrait;
+
+class EntityManagerTest extends TestCase
+{
+    public function testLazyGhostEntityManager(): void
```

Comment:
https://github.com/doctrine/orm/issues/5933 should probably be mentioned here, in one form or another.

Reply:
I've added a comment about this